test/unit/lib/plugins/variables.test.js

```diff
@@ -0,0 +1,79 @@
+const { when } = require('jest-when')
+const Variables = require('../../../../lib/plugins/variables')
+
+describe('Variables', () => {
+  let github
+  const org = 'bkeepers'
+  const repo = 'test'
+
+  function fillVariables (variables = []) {
+    return variables
+  }
+
+  function configure (config) {
```

Comment:
The config argument is not used. Can we remove it?

Reply:
done